src/main.rs

```diff
@@ -57,12 +57,18 @@ async fn readyz() -> Result<impl Responder, Error> {
 async fn mail(
     request: HttpRequest,
     body: web::Bytes,
-    mailer: web::Data<SmtpTransport>,
+    state: web::Data<State>,
```

Comment:
Hvorfor trenger vi Data<State>

Reply:
Den inneholder mail client og allowlist, så man slipper å lage det for hvert kall.